netket/optimizer/sr/sr_onthefly_logic.py

```diff
@@ -91,16 +40,32 @@ def O_vjp(x, params, v, forward_fn):
     return jax.tree_map(sum_inplace, res)  # allreduce w/ MPI.SUM
 
 
-def O_mean(samples, params, forward_fn):
+def O_mean(samples, params, forward_fn, holomorphic=True):
```

Comment:
can you put arguments in the order we use normally? `forward_fun, params, samples` (a bit everywhere